dev/com.ibm.websphere.appserver.features/visibility/public/jakartaMail-1.6/com.ibm.websphere.appserver.jakartaMail-1.6.feature

```diff
@@ -6,6 +6,6 @@ IBM-ShortName: jakartaaMail-1.6
 Subsystem-Version: 1.6
 Subsystem-Name: JakartaMail 1.6
 -bundles=\
-  com.ibm.websphere.jakarta.activation.2.0; location:="dev/api/spec/,lib/";mavenCoordinates="jakarta.activation:jakarta.activation-api:2.0"
+  io.openliberty.activation.2.0; location:="dev/api/spec/,lib/";mavenCoordinates="jakarta.activation:jakarta.activation-api:2.0"
```

Comment:
This doesn't make sense.  Why is this feature called jakartaMail-1.6?  Why isn't it called 2.0?  The feature name should be io.openliberty.jakarta.activation.2.0.

Reply:
I assume you are going to leave the JakartaMail version alone and let Mike change it to 2.0?

Reply:
Correct, since he's actively working on the feature I'm going to pass it on to him to update it. @uberskigeek FYI